setup/emc-update-impl.bash

```diff
@@ -124,7 +124,7 @@ fi
 _make_sure_installed python3-catkin-tools libassimp-dev ros-"${EMC_ROS_DISTRO}"-cv-bridge ros-"${EMC_ROS_DISTRO}"-image-geometry ros-"${EMC_ROS_DISTRO}"-map-server ros-"${EMC_ROS_DISTRO}"-message-generation ros-"${EMC_ROS_DISTRO}"-message-runtime ros-"${EMC_ROS_DISTRO}"-nav-msgs ros-"${EMC_ROS_DISTRO}"-roscpp ros-"${EMC_ROS_DISTRO}"-shape-msgs ros-"${EMC_ROS_DISTRO}"-tf2
 
 # 4) Compile
-catkin build --workspace "$EMC_SYSTEM_DIR"
+catkin build --workspace "$EMC_SYSTEM_DIR" --no-status
```

Comment:
I do think you want the status outside CI. So this should be conditional based on CI env variabele.